canonical-websites/www.ubuntu.com
# 2753: Build openstack/install/try-openstack page

## Done

- Moved /download/cloud/try-openstack to /openstack/install/try-openstack
- Updated copy according to [copydoc](https://docs.google.com/document/d/1QIXOAlCDV2cyQqU5uEW2kpITobf7OkFHr-tsEvjd_1Q/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/install/try-openstack
- Check that it matches the [copydoc](https://docs.google.com/document/d/1QIXOAlCDV2cyQqU5uEW2kpITobf7OkFHr-tsEvjd_1Q/edit#)
- Check that /download/cloud/try-openstack redirects to /openstack/install/try-openstack

## Issue / Card

Fixes #2753
